tests/LSP14Ownable2Step/LSP14Ownable2Step.behaviour.ts

```diff
@@ -433,13 +433,17 @@ export const shouldBehaveLikeLSP14 = (
       });
 
       describe("when called after the delay and before the confirmation period end", () => {
-        it("should have emitted a OwnershipTransferred event", async () => {
+        beforeEach(async () => {
+          // Call renounceOwnership for the first time
           await context.contract
             .connect(context.deployParams.owner)
             .renounceOwnership();
 
-          await network.provider.send("hardhat_mine", ["0x63"]); // skip 99 blocks
+          // Skip 199 block to reach the time where renouncing ownership can happen
+          await network.provider.send("hardhat_mine", ["0xC7"]); //
```

Comment:
can't we put the number in decimals? for readability and to remove the comment

```suggestion
          await network.provider.send("hardhat_mine", [199]);
```

otherwise, maybe we can use `BigNumber.from(199).toHex()`

If any of these work.